plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -183,7 +183,7 @@ func ClusterRoles() []rbac.ClusterRole {
 
 				rbac.NewRule(ReadWrite...).Groups(batchGroup).Resources("jobs", "cronjobs").RuleOrDie(),
 
-				rbac.NewRule(ReadWrite...).Groups(extensionsGroup).Resources("daemonsets",
+				rbac.NewRule(ReadWrite...).Groups(extensionsGroup, appsGroup).Resources("daemonsets",
 					"deployments", "deployments/scale", "deployments/rollback", "ingresses",
 					"replicasets", "replicasets/scale", "replicationcontrollers/scale").RuleOrDie(),
```

Comment:
replicationcontrollers/scale is not in the appsGroup... I would add the replicasets and replicasets/scale resources to the appsGroup rule on line 179 instead, alongside deployments